docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
```

Comment:
I saw @bclermont too late, to reply in the same comment.

Didn't thought about `fuse`. Should be fine to use `alpine` rather than `scratch` then :)